LICENSE

```diff
@@ -0,0 +1,20 @@
+Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)
+Copyright (C) 2021 DEVSTRONS' and contributors
```

Comment:
Actually, I think it'd be better like this. 😅
```suggestion
Copyright (C) 2021 DEVSTRONS'
```

Reply:
Made the necessary changes.